### PR TITLE
DDF-5743 Added new property for solr downloads

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Adds a new property for Solr Download URLs

#### Who is reviewing it? 

@AdamBurstynski 
@LinkMJB 
@jrnorth 

#### Ask 2 committers to review/merge the PR and tag them here.
@pklinef
@AzGoalie 

#### How should this be tested?

Full build passes

#### What are the relevant tickets?
Fixes: #5743 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/5744)

### CODE REVIEW

File: distribution/solr-distro/pom.xml [hunk 1]
```diff
@@ -99,12 +99,13 @@
                         </goals>
                         <configuration>
                             <url>
-                                http://artifacts.codice.org/content/repositories/thirdparty/org/apache/archive/solr/${solr.version}/solr-${solr.version}.zip
```
Comment (pklinef) [inline distribution/solr-distro/pom.xml]: :question: Why 7.4 instead of solr.version which is currently 7.7.2?

Comment (shaundmorris) [reply distribution/solr-distro/pom.xml]: I thought it was time for a downgrade :-) I'll change it to 7.7.2

Comment (shaundmorris) [reply distribution/solr-distro/pom.xml]: Actually meant to keep the property ${solr.version} there

**Comment by shaundmorris** [comment]
build now

File: distribution/solr-distro/pom.xml [hunk 2]
```diff
@@ -99,12 +99,13 @@
...
+                              ${solr.download.url}/lucene/solr/${solr.version}/solr-${solr.version}.zip
                             </url>
                             <unpack>true</unpack>
                             <failOnError>true</failOnError>
-                            <sha512>aca27ff6942872a571160f49f5e4d43090dbe9bb37da0d1999557e1655f25055173ef999bfb8eba8f19f19b9c438b43205c805d0d294b9756cf5c0aaf66a656a</sha512>
-                            <checkSignature>true</checkSignature>
+                            <!--temporarily disable check so the zip can be obtained from multiple sources
```
Comment (pklinef) [inline distribution/solr-distro/pom.xml]: The SHA512 checksum for Solr 7.7.2 should always be the same regardless of source.

:question: Should the SHA512 be a property too?

Comment (shaundmorris) [reply distribution/solr-distro/pom.xml]: Looks like it works now I was probably doing something else wrong so removed this section.

**Comment by shaundmorris** [comment]
build now

running the build again after latest commit. looks like itests didn't run.

**Comment by shaundmorris** [comment]
build now

Verifying if the 9 itest failed due to this change.

**Comment by shaundmorris** [comment]
I figured out why those itests fail. They depend on DDF SDK artifacts that are not currently accessible so this build is fine and need to fix that as a separate issue.

**Comment by shaundmorris** [comment]
Merging to move PR builds forward. May re-visit the default value pending artifact repositories in the future.

**Pull request merged**